theme/js/components/dataset/search-result.vue

```diff
@@ -45,13 +45,68 @@
             {{ $filters.excerpt(description, 160) }}
           </p>
           <p class="fr-mb-0 text-mention-grey">
+            <Tooltip class="fr-hidden inline-sm">
+                <template #tooltip>
+                  <h5 class="fr-text--sm fr-my-0">{{$t("Metadata quality:")}}</h5>
+                  <QualityItem
+                    :passed="quality.dataset_description_quality"
+                    :messagePassed='$t("Data description filled")'
+                    :messageFailed='$t("Data description empty")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.license"
+                    :messagePassed='$t("License filled")'
+                    :messageFailed='$t("No license set")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.update_frequency && quality.update_fulfilled_in_time"
+                    :messagePassed='$t("Update frequency followed")'
+                    :messageFailed='quality.update_fulfilled_in_time ? $t("Update frequency not followed") : $t("Update frequency not set")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.has_open_format"
+                    :messagePassed='$t("File formats are open")'
+                    :messageFailed='$t("File formats are closed")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.temporal_coverage"
+                    :messagePassed='$t("Temporal coverage filled")'
+                    :messageFailed='$t("Temporal coverage not set")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.spatial_coverage"
```

Comment:
This criterion is called `spatial` only.
```suggestion
                    :passed="quality.spatial"
```